src/app/globals.css

```
@@ -1173,11 +2003,123 @@ h1, h2, h3, h4, h5, h6 {
+}
+.prose-page__body ul,
+.prose-page__body ol {
+  padding-left: 1.5rem;
+  margin: 0 0 16px;
+}
+.prose-page__body ul li::marker { color: var(--color-mid-gray); }
+.prose-page__body strong { color: var(--color-primary); font-weight: 600; }
+.prose-page__body a {
+  color: var(--color-brand);
+  text-decoration: none;
+  border-bottom: 1px solid currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)